src/domain/challenge/ecoverse/ecoverse.resolver.mutations.ts

```diff
@@ -6,18 +6,33 @@ import { Resolver, Args, Mutation } from '@nestjs/graphql';
 import { GqlAuthGuard } from '@src/core/authorization/graphql.guard';
 import { Roles } from '@common/decorators/roles.decorator';
 import { Profiling } from '@src/common/decorators';
-import { UpdateEcoverseInput } from './ecoverse.dto.update';
-import { Ecoverse } from './ecoverse.entity';
-import { IEcoverse } from './ecoverse.interface';
 import { EcoverseService } from './ecoverse.service';
 import { AuthorizationRoles } from '@src/core/authorization/authorization.roles';
+import {
+  CreateEcoverseInput,
+  Ecoverse,
+  IEcoverse,
+  UpdateEcoverseInput,
+} from '@domain/challenge/ecoverse';
 
 @Resolver()
 export class EcoverseResolverMutations {
   constructor(
     @Inject(EcoverseService) private ecoverseService: EcoverseService
   ) {}
 
+  @Roles(AuthorizationRoles.EcoverseAdmins)
```

Comment:
I don't think EcoverseAdmin should be able to create an Ecoverse - his scope is inside the Ecovse, not on the 'Multiverse' scope - only Global Admin should have authority there in my opinion.

Reply:
Agree.